extide/gradle/netbeans-gradle-tooling/src/main/java/org/netbeans/modules/gradle/tooling/NbProjectInfoBuilder.java

```diff
@@ -811,14 +885,23 @@ private void inspectExtensions(String prefix, ExtensionContainer container) {
             
             Object ext;
             try {
-                ext = project.getExtensions().getByName(extName);
+                ext = container.getByName(extName);
```

Comment:
this was a bug !! The extension enumerated from a domain object was always looked up as extension of a project itself.